src/jmh/java/testing/SampleBenchmark.java

```diff
@@ -0,0 +1,25 @@
+package testing;
```

Comment:
We could put this test into separated testing folder called performance

Reply:
its jmh standard to put it to src/jmh and I did not find how to change it